target/linux/generic/backport-5.4/716-v5.5-net-sfp-move-fwnode-parsing-into-sfp-bus-layer.patch

```diff
@@ -42,6 +42,10 @@ Signed-off-by: Russell King <rmk+kernel@armlinux.org.uk>
  		return ret;
  	}
  
+@@ -587,9 +580,7 @@ static int phylink_register_sfp(struct p
+ 		return 0;
+ 	}
+ 
```

Comment:
commit https://github.com/gregkh/linux/commit/ff19d70b665d43a90081b7a044e61db051e223cf being part of 5.4.176 made this manual rebase necessary, and it looks good to me.